spec/spec_helper.rb

```diff
@@ -87,3 +87,7 @@
   #   # as the one that triggered the failure.
   #   Kernel.srand config.seed
 end
+
+def in_ci_mode?
```

Comment:
@jenlindner see comment below. You don't need an additional method to determine ci.